doc/classes/PhysicsDirectSpaceState3D.xml

```diff
@@ -21,7 +21,7 @@
 			</description>
 		</method>
 		<method name="collide_shape">
-			<return type="PackedVector2Array[]" />
+			<return type="PackedVector3Array" />
```

Comment:
Oh this seems wrong, it will likely fail CI.
It's a typed array of PackedVector3Array, so the `[]` is part of the type.

Reply:
Fixed!

Reply:
Do you know if `void*` should be updated in [PhysicsDirectSpaceState3DExtension](https://github.com/godotengine/godot/blob/master/doc/classes/PhysicsDirectSpaceState3DExtension.xml#L34) or is that unrelated?

Reply:
I think it's fine, it's a pointer to the results, whose type now changed. So this will likely break code using this in GDExtension, but that's necessary to fix this bug.